src/zui/ZUIMenu/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { makeStyles } from '@mui/styles';
+import { FC, ReactNode } from 'react';
+import {
+  ListItemIcon,
+  ListItemText,
+  Menu,
+  MenuItem,
+  Theme,
+  Typography,
+} from '@mui/material';
+
+const useStyles = makeStyles<Theme, { maxHeight?: string; width?: string }>(
+  (theme) => ({
+    menu: {
+      '& ul': {
+        maxHeight: ({ maxHeight }) => (maxHeight ? maxHeight : ''),
+        width: ({ width }) => (width ? width : ''),
+      },
+    },
+    paper: {
+      boxShadow: theme.elevation.bottom.small.medium,
+    },
+  })
+);
+
+export interface MenuItem {
+  disabled?: boolean;
+  divider?: boolean;
+  endContent?: ReactNode;
+  label: string;
+  onClick: () => void;
+  startIcon?: ReactNode;
+}
+
+interface ZUIMenuProps {
+  anchorEl?: Element | null;
+  dense?: boolean;
+  disableGutters?: boolean;
+  menuItems: MenuItem[];
+  maxHeight?: string;
+  onClose?: () => void;
+  smallScreen?: boolean;
```

Comment:
Not sure if this should be a property, or a media query inside of this component. I believe the latter makes more sense to me. What do you think?

Reply:
this is simply "translated" from the properties that were described in the figma design. I thought it was a bit weird to, but just went with it - i think removing it for now and implementing it when we actually do work to make the design system good for small screens feels better?

Reply:
Yeah, I agree. You should not interpret the properties in figma as properties for React. They are in-figma settings that the designer can use for the figma component. This case is a good example of where something might be necessary in figma but not relevant for code.